Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN test -z "$(gofmt -l $(find . -type f -name '*.go' -not -path "./vendor/*"))"
 # ldflags -X injects commit version into binary
 
 RUN license-check -path ./ --verbose=false \
- && go test $(go list ./... | grep -v /vendor/ | grep -v /template/|grep -v /build/) -cover \
+ && OPEN_FAAS_TELEMETRY=0 go test $(go list ./... | grep -v /vendor/ | grep -v /template/|grep -v /build/) -cover \
```

Comment:
Is it easier to put this as an ENV just above?

Reply:
Hmm, could do.